parser_library/src/parsing/parser_impl.cpp

```diff
@@ -119,6 +120,12 @@ context::id_index parser_impl::parse_identifier(std::string value, range id_rang
     return hlasm_ctx->ids().add(std::move(value));
 }
 
+size_t parser_impl::get_loctr_len() const
+{
+    auto [_, opcode] = *proc_status;
+    return processing::processing_status_cache_key::generate_loctr_len(opcode.value);
```

Comment:
Is the processing status/opcode valid when parsing something like
```
&VARP(L'=CL(L'*)'1') SETA 1
```

How does the assembler handle `L'*` in labels?

Reply:
It currently does not, but IBM has released an APAR PH42377 that enhances parsing of literal attributes, so this will have to be verified again once we get it on our systems.
I'll add a test and fix the problem.